analyzers/src/SonarAnalyzer.CSharp/Rules/RedundantInheritanceList.cs

```diff
@@ -199,14 +173,17 @@ private static bool CanInterfaceBeRemovedBasedOnMembers(INamedTypeSymbol declare
             foreach (var interfaceMember in allMembersOfInterface)
             {
                 var classMember = declaredType.FindImplementationForInterfaceMember(interfaceMember);
-                if (classMember != null
+                if (classMember is not null
```

Comment:
To remove the standalone declaration
```suggestion
                if (declaredType.FindImplementationForInterfaceMember(interfaceMember) is { } classMember
```